ubuntuStation/forms.py

```diff
@@ -25,18 +25,8 @@ def __call__(self, form, field):
             #print("Error")
             raise ValidationError('Entry with this value already exists in the database')
 
-'''
 class QuestionForm(FlaskForm):
     questiontext = TextAreaField('questiontext', validators=[DataRequired()], description=u'Question')
-    questionurl = TextField('questionurl', validators=[DataRequired()], description=u'Question URL')
-'''
-'''
-class ActiveQuestionForm(FlaskForm):
```

Comment:
Yeah you are right, ActiveQuestion is a part of the other commit.   I send a new PR in a bit, hopefully it should be the last one for this issue.